sdk/eidolon_ai_sdk/agent/simple_agent.py

```diff
@@ -96,9 +97,10 @@ class SimpleAgentSpec(BaseModel):
     system_prompt: str = "You are a helpful assistant"
     agent_refs: List[str] = []
     actions: List[ActionDefinition] = [ActionDefinition()]
-    cpu: AnnotatedReference[AgentCPU] = None
```

Comment:
we should change the key from cpu to apu. leave cpu, use it to set apu if it is provided, and log a deprecated warning.

Reply:
Done